src/Surfnet/StepupMiddlewareClient/Identity/Dto/RaCandidateSearchQuery.php

```diff
@@ -67,13 +72,15 @@ class RaCandidateSearchQuery implements HttpQuery
      * @param string $institution
      * @param int    $pageNumber
      */
-    public function __construct($actorInstitution, $pageNumber)
+    public function __construct($actorId, $actorInstitution, $pageNumber)
```

Comment:
Please add the `actorId` to the parameters in the doc block

Reply:
`@param string $institution` should also be updated to read:
`@param string $actorInstitution`